Userland/Utilities/image.cpp

```diff
@@ -23,6 +23,9 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
     StringView out_path;
     args_parser.add_option(out_path, "Path to output image file", "output", 'o', "FILE");
 
+    bool ascii_ppm;
+    args_parser.add_option(ascii_ppm, "Convert to a PPM in ASCII", "ppm-ascii", {});
```

Comment:
Please keep the option name and the underlying variable the same (ppm-ascii vs ascii-ppm)